Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,38 @@
 
 namespace Hazel {
 
+	template <size_t N>
+	struct CleanExpressionResult
+	{
+		char Data[N];
+	};
+
+	template <size_t N>
+	constexpr auto CleanExpression(const char(&expr)[N])
```

Comment:
Make an equivalent function `ReplaceQuotes` for inside the `HZ_PROFILE_SCOPE` macro, so we can remove these lines
https://github.com/TheCherno/Hazel/blob/47126d585ae56ceebb3fdeae1da635d1c240b02e/Hazel/src/Hazel/Debug/Instrumentor.h#L113-L114

Reply:
Done.